app/webpack/src/success_prospects.js

```diff
@@ -0,0 +1,17 @@
+$(function() {
```

Comment:
I think this needs to be rewritten in vanilla JS not using jQuery otherwise Rose will have to redo it.

Reply:
^good point although actually this file has been deleted in AP-1821 which was merged last Tuesday, if you rebase off master then there is no need to update this :)

Reply:
@eloisetait3 just tagging you in case you don't get notifications for single comments

Reply:
That's what I thought. Cool.

Reply:
@RoseSAK Just rebased on master but the file is still there 😕 Should I just manually delete the file?